### PR TITLE
Added warning message if terms checkbox deleted intentionally

### PR DESCRIPTION
**Pull request by vermadarsh**
I added a piece of code in JS to restrict post updates if the HTML entities are deleted intentionally.
Issue: #1146

### CODE REVIEW

File: app/assets/js/rtm-upload-terms.js [hunk 1]
```diff
@@ -1,139 +1,169 @@
...
+            jQuery( '.rt_alert_msg' ).remove();
+        }
+
+        return true;
+    });
+
+    /**
+     * Uncheck the terms checkbox after the activity is posted successfully.
+     */
+    rtMediaHook.register( 'rtmedia_js_after_activity_added', function () {
```
Comment (SID177) [inline app/assets/js/rtm-upload-terms.js]: Don't we have to `return true` in this function?

File: app/assets/js/rtm-upload-terms.js [hunk 2]
```diff
@@ -1,139 +1,169 @@
...
+jQuery( document ).ready( function () {
+
+    var terms_conditions_checkbox = jQuery( '#rtmedia_upload_terms_conditions' );
+    if ( 1 === terms_conditions_checkbox.length ) {
+        terms_conditions_checkbox.on( 'click', function () {
+
+            // If start upload button exist, then focus to that button.
+            var upload_start_btn = jQuery( '.start-media-upload' );
+            if ( 1 === upload_start_btn.length ) {
+                upload_start_btn.focus();
```
Comment (SID177) [inline app/assets/js/rtm-upload-terms.js]: Just write `jQuery( '.start-media-upload' ).focus()`.

File: app/assets/js/rtm-upload-terms.js [hunk 3]
```diff
@@ -1,139 +1,169 @@
...
+
+        return true;
+    });
+
+    /**
+     * Uncheck the terms checkbox after the activity is posted successfully.
+     */
+    rtMediaHook.register( 'rtmedia_js_after_activity_added', function () {
+        var rtmedia_terms_conditions = $( '#rtmedia_upload_terms_conditions' );
+        if ( 1 === rtmedia_terms_conditions.length && rtmedia_terms_conditions.is( ':checked' ) ) {
```
Comment (SID177) [inline app/assets/js/rtm-upload-terms.js]: No need to check any condition, just uncheck the checkbox.
Refer - https://stackoverflow.com/questions/38751168/do-i-really-need-to-check-if-an-element-exists-with-jquery#answer-38751351

File: app/assets/js/rtm-upload-terms.js (hunk 2 above)
Comment (vermadarsh) [reply app/assets/js/rtm-upload-terms.js]: `upload_start_btn` variable is used in line 142.

File: app/assets/js/rtm-upload-terms.js [hunk 4]
```diff
@@ -1,139 +1,163 @@
...
+                whats_new_submit.removeAttr( 'disabled' );
+                whats_new_submit.removeClass( 'loading' );
+
+                return args;
+            }
+
+            if ( ! terms_conditions_checkbox.is( ':checked' ) ) {
+                whats_new_submit.removeAttr( 'disabled' );
+                whats_new_submit.removeClass( 'loading' );
+                if ( args && 'activity' === args.src ) {
```
Comment (SID177) [inline app/assets/js/rtm-upload-terms.js]: Leave blank line above.

File: app/assets/js/rtm-upload-terms.js [hunk 5]
```diff
@@ -1,139 +1,163 @@
...
+
+    /**
+     * When Select Attribute for media [ rtmedia-custom-attributes: Add-Ons ] Issue:8,
+     * This Hook returns false that's why the attributes were not saved,
+     * Added: 'return true;'
+     * By: Yahil
+     */
+    rtMediaHook.register( 'rtmedia_js_after_file_upload', function () {
+        var terms_conditions_checkbox = jQuery( '#rtmedia-upload-container #rtmedia_upload_terms_conditions' );
+        if ( 1 === terms_conditions_checkbox.length ) {
```
Comment (SID177) [inline app/assets/js/rtm-upload-terms.js]: Blank line above.

File: app/assets/js/rtm-upload-terms.js [hunk 6]
```diff
@@ -1,139 +1,163 @@
...
+    terms_conditions_checkbox.on( 'click', function () {
+
+        // Focus on `start upload` button.
+        var upload_start_btn = jQuery( '.start-media-upload' );
+        upload_start_btn.focus();
+
+        // Show error message if terms-condition is not checked.
+        if ( terms_conditions_checkbox.is( ':checked' ) ) {
+            var alter_msg_span = terms_conditions_checkbox.siblings( 'span.rt_alert_msg' );
+            if ( 1 === alter_msg_span.length ) {
```
Comment (SID177) [inline app/assets/js/rtm-upload-terms.js]: Blank line above.

File: app/assets/js/rtm-upload-terms.js [hunk 7]
```diff
@@ -142,14 +166,18 @@ function rtp_display_terms_warning ( selector, rt_alert_msg ) {
...
-		});
-	});
-}
-
-jQuery( document ).ready( function() {
-	handle_privacy_message();
-} );
+    jQuery( '#close_rtm_privacy_message' ).on( 'click', function (c) {
+        var privacy_wrapper = jQuery( '.privacy_message_wrapper' );
+        privacy_wrapper.fadeOut( 'slow', function () {
```
Comment (SID177) [inline app/assets/js/rtm-upload-terms.js]: Blank line above.

File: app/assets/js/rtm-upload-terms.js (hunk 4 above)
Comment (vermadarsh) [reply app/assets/js/rtm-upload-terms.js]: @SID177 there is already a blank space before `if` statement.

**Comment by vermadarsh** [comment]
@SID177 
changes applied, please review.

**Pull request merged**